flow/activities/flowable.go

```diff
@@ -841,94 +814,38 @@ func (a *FlowableActivity) AddTablesToPublication(ctx context.Context, cfg *prot
 	return err
 }
 
-func (a *FlowableActivity) LoadPeer(ctx context.Context, peerName string) (*protos.Peer, error) {
-	row := a.CatalogPool.QueryRow(ctx, `
-		SELECT name, type, options
-		FROM peers
-		WHERE name = $1`, peerName)
+// TODO remove in 0.15
```

Comment:
don't remove, could be used for handling config changes from UI

Reply:
must not be called from temporal, UI endpoints must update directly